common/spec/dependabot/dependency_group_spec.rb

```diff
@@ -117,17 +117,17 @@
       context "when no dependencies are assigned to the group" do
         it "returns true if the dependency matches a pattern" do
           expect(dependency_group.dependencies).to eq([])
-          expect(dependency_group.contains?(test_dependency1)).to be_truthy
+          expect(dependency_group).to be_contains(test_dependency1)
         end
 
         it "returns false if the dependency is specifically excluded" do
           expect(dependency_group.dependencies).to eq([])
-          expect(dependency_group.contains?(test_dependency2)).to be_falsey
+          expect(dependency_group).not_to be_contains(test_dependency2)
```

Comment:
was this supposed to be `includes`?

Reply:
this is what the autocorrect did to the code.  dependency_group has a method defined - contains? which wil return a boolean.  The strict version of the cop says "to be_truthy" and "to be_falsey" are bad.  So, it tries to rewrite the test condition.

The implementation of contains? is as follows:

    sig { params(dependency: Dependabot::Dependency).returns(T::Boolean) }
    def contains?(dependency)
      return true if @dependencies.include?(dependency)
      return false if matches_excluded_pattern?(dependency.name)

      matches_pattern?(dependency.name) && matches_dependency_type?(dependency)
    end
This will only return true or false, so be_true or be_false would work fine

Reply:
include? only works on arrays or other groups, not on objects.  dependency_group is an object.

Reply:
fixed with latest commit